quality/metrics/metrics.md

```diff
@@ -0,0 +1,56 @@
+---
+parent: Building and research
+has_children: true
+nav_order: 150
+title: Metrics
+description: Metrics for evaluating machine translation
+---
+
+Metrics are formulas that represent the quality of the machine translation output.
+
+Machine translation quality metrics are divided into string-based metrics and pre-trained metrics.
+
+## String-based metrics
+
+String-based metrics measure the lexical matching between the system output and the reference translation.
+
+### Challenges
+
+String-based metrics are challenging because there are many possible translations for a single source text.
+
+### Examples
+
+- BLEU
+- METEOR
+- NIST
+- chrF
+- TER
+
+*Note: The list is incomplete.*
+
+### Pre-trained metrics
```

Comment:
I'd call these "Machine learning metrics", "Machine learning-based metrics",  "Predictive metrics", "Model-based metrics" or something else.

"Pre-trained" is too narrow here, because COMET, for example, is also fine-tuned.

Reply:
Also, this should be ## not ###